index.css

```diff
@@ -1,3 +1,3 @@
 h1{
```

Comment:
[ ] kindly use a CSS class to style your header, it's part of the project requirements. kindly fix that 👇🤓
![css](https://user-images.githubusercontent.com/62156607/204811675-eddb41c2-aaea-4802-ae51-ad52b96b3bdf.PNG)